modules/sdk-coin-sei/test/integration/index.ts

```diff
@@ -0,0 +1,16 @@
+import 'should';
+
+import { BitGoAPI } from '@bitgo/sdk-api';
+import { TestBitGo, TestBitGoAPI } from '@bitgo/sdk-test';
+
+import { Sei, Tsei } from '../../src/index';
+
+describe('SEI', function () {
```

Comment:
it doesn't have any testcase?

Reply:
Yes, But other sdks have this kind of placeholder for integration tests. May be if we want to add integration tests in future.